packages/components/src/Checkbox/Checkbox.stories.tsx

```diff
@@ -0,0 +1,122 @@
+import type { StoryObj } from "@storybook/react";
+import React from "react";
+import EDSCheckbox, { CheckboxInput, Label } from "./Checkbox";
+
+const defaultArgs = {
+  disabled: false,
+  label: "Checkbox",
+};
+
+/**
+ * Controlled example to make checked stories interactive.
+ * We name this "Checkbox" for Storybook's code documentation
+ */
+function Checkbox({ checked: defaultChecked = false, ...rest }: Args) {
```

Comment:
Made this the default component for Storybook b/c I liked what @jinlee93 did in the [Toggle](https://github.com/FB-PLP/traject/blob/master/app/assets/javascripts/v2/core/EDSCandidates/Toggle/Toggle.stories.tsx#L9)

Reply:
Another thing to note is Storybook is having issues parsing the props in either case 😬 

Now, it only shows props in `defaultArgs` or `argTypes` and doesn't load descriptions:
<img src="https://user-images.githubusercontent.com/15840841/149577811-07845ad2-90aa-48e1-86ad-e532531c43d4.png" width="400">

Previously, it loaded more props but didn't include `size` and marked `id` as required (which isn't true for the overall component):
<img src="https://user-images.githubusercontent.com/15840841/149582492-95b7b99b-7d28-4378-92bd-11ec61162ec2.png" width="600">

I'll make a ticket to dig into this more, maybe we're doing something wrong with the docgen config

Reply:
I also wonder if react-docgen-typescript expects to find types/descriptions associated with the component itself, and not passed through an intermediary.